src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryStep.java

```diff
@@ -157,15 +161,20 @@ public AutoCompletionCandidates doAutoCompleteIdentifier(@AncestorInPath ItemGro
 
     }
 
-    public static class Execution extends AbstractSynchronousNonBlockingStepExecution<LoadedClasses> {
+    public static class Execution extends SynchronousNonBlockingStepExecution<LoadedClasses> {
 
         private static final long serialVersionUID = 1L;
 
-        @Inject private transient LibraryStep step;
-        @StepContextParameter private transient Run<?,?> run;
-        @StepContextParameter private transient TaskListener listener;
+        private transient final LibraryStep step;
```

Comment:
Somewhat oxymoronic field modifiers, but a `SynchronousNonBlockingStepExecution` is never usefully serialized so it makes no difference.